Wikipedia/UI-V5/WMFSaveableTitleCollectionViewCell.h

```diff
@@ -0,0 +1,34 @@
+
+#import <SSDataSources/SSDataSources.h>
+
+@class MWKTitle;
+@class MWKSavedPageList;
+@class MWKImage;
+
+/**
+ *  Base class for cells which represent a title and have save button.
+ *
+ *  Will also be given the "card" style by applying a white background to its
+ *  @c contentView as well as a small shadow.
+ */
+@interface WMFSaveableTitleCollectionViewCell : SSBaseCollectionCell
```

Comment:
Feels like the name should be a little more generic since this class includes 2 distinct pieces of functionality:

- Saving
- Shadow applying

Or, if you prefer to be more specific rather than generic, you may consider leaving "WMFShadowCell" as it's own subclass and having `WMFSaveableTitleCollectionViewCell` inherit from that instead.